### ISSUE TITLE
jspm-resolve namespace

### ISSUE THREAD

**Issue by guybedford** [nfl/jspm-resolve#11]
I'd like to make a `npm install jspm-resolve` an officially supported package for jspm 2.

Would there be any interest in transferring ownership here?

I'd be sure to publish from the 3.x version to avoid breaking existing semver contracts in installs.

**Comment by potench** [comment]
Do you just need the NPM namespace for `jspm-resolve` or do we need to transfer the whole git project? https://help.github.com/articles/about-repository-transfers/

**Comment by guybedford** [comment]
Yeah just the npm access, if you don't still need the 'latest' tag. Otherwise I can publish under jspm-resolver, but thought I'd check here first.
